packages/manager/apps/catalog-revamp/src/components/Filters/FilterItem.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { OsdsText } from '@ovhcloud/ods-components/text/react';
+import { OsdsCheckbox } from '@ovhcloud/ods-components/checkbox/react';
+import { OsdsCheckboxButton } from '@ovhcloud/ods-components/checkbox-button/react';
+import { OsdsSkeleton } from '@ovhcloud/ods-components/skeleton/react';
+import { ODS_TEXT_LEVEL, ODS_TEXT_SIZE } from '@ovhcloud/ods-components/text';
+import { ODS_CHECKBOX_BUTTON_SIZE } from '@ovhcloud/ods-components/checkbox-button';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import './Filters.scss';
+
+interface FilterItemProps {
+  label?: string;
+  count?: number;
+  loading?: boolean;
+  type: 'category' | 'universe' | 'loading';
+  lineNumber?: number;
+  isChecked?: boolean;
+  onCheckboxChange?: (
+    type: 'category' | 'universe',
+    label: string,
+    event: CustomEvent,
+  ) => void;
+}
+
+const FilterItem: React.FC<FilterItemProps> = ({
+  label,
+  count,
+  loading,
+  type,
+  lineNumber = 1,
+  isChecked,
+  onCheckboxChange,
+}) => {
+  const resultsNumber = count ? ` (${count})` : '';
+
+  if (type === 'loading') {
+    return (
+      <>
+        {Array.from({ length: lineNumber }, (_, index) => (
+          <div key={index} className="break-inside-avoid">
+            <OsdsSkeleton />
+          </div>
+        ))}
+      </>
+    );
+  }
```

Comment:
Do you really need to put that in the FilterItem component ?
Can't it be a distinct component ?

Reply:
In a way this is a filterItem which is loading. What do you have in mind ? You wanted to use the loading component used for the results or create another one ?

Reply:
Done.